### ISSUE TITLE
Dedicated branch for Pull Requests

### ISSUE THREAD

**Issue by w-gomes** [TheCherno/Hazel#54]
Hello.
There will be definitely more pull requests as the project grows. I think it looks cleaner just to have a dedicated branch for pull requests. Then, later on, it can be merged into master.

Maybe call it `nightly` or something.

Cheers.

**Comment by IronicallySerious** [comment]
I think this repo will be forked and then made read-only after the series comes to a halt. The newer fork can be put out for being further developed by the community

**Comment by LovelySanta** [comment]
As the project is growing, having 1 commit/PR isn't that bad, and it is also handy to look back at the commits to see what happened between the episodes on the series. Closing this for now, if we need it in the future, we can always add a new branch. For big features, of course we can still create new branches until it is fine tuned, or when the series come to a halt (touch wood, hopefully not any time soon).
